# An array of Local Goals with a header for global reference
Header header
user_intentions/LocalGoal[] goals
